### DISCUSSION TITLE
"Index object count mismatch." What should I do?

### DISCUSSION THREAD

**Discussion by fain182** [borgbase/vorta#1847] [FAQ] [answered]
I made a backup and an error message appeared:

```
Borg exited with warning status (rc 1). See the [logs](file:///home/fain182/.local/state/Vorta/log) for details.
Remote: Index object count mismatch.
```

What should I do? I saw that there is a `check --repair` command in borg, there is a button for it in vorta?

**Comment by m3nu** [comment, marked as answer]
No, there is no button for this in Vorta, since the command is interactive and may require additional action. So better run it in the CLI.

On the bright side, this error is often harmless.

**Reply by fain182** [reply]
If I run `borg check` the output is:
`borg check: error: argument REPOSITORY_OR_ARCHIVE: Invalid location format: ""`

I should give the remote repository url?

**Reply by m3nu** [reply]
Like for every command, you will need to provide the correct arguments, yes. 😬

Repo URL can be copied from Vorta. We have a button for it. You'll also need the repo password.
